doc/ref_arch/openstack/chapters/chapter01.rst

```diff
@@ -1,167 +1,167 @@
-
-OpenStack specific principles
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-OpenStack considers the following Four Opens essential for success:
-
--  Open Source
--  Open Design
--  Open Development
--  Open Community
-
-This OpenStack Reference Architecture is organised around the three
-major Cloud Infrastructure resource types as core services of compute,
-storage and networking, and a set of shared services of identity
-management, image management, graphical user interface, orchestration
-engine, etc.
-
-Document Organisation
----------------------
-
-Chapter 2 defines the Reference Architecture requirements and, when
-appropriate, provides references to where these requirements are
-addressed in this document. The intent of this document is to address
-all of the mandatory (“must”) requirements and the most useful of the
-other optional (“should”) requirements. Chapter 3 and 4 cover the Cloud
-Infrastructure resources and the core OpenStack services, while the APIs
-are covered in Chapter 5. Chapter 6 covers the implementation and
-enforcement of security capabilities and controls. Life Cycle Management
-of the Cloud Infrastructure and VIM are covered in Chapter 7 with stress
-on Logging, Monitoring and Analytics (LMA), configuration management and
-some other operational items. Please note that Chapter 7 is not a
-replacement for the implementation, configuration and operational
-documentation that accompanies the different OpenStack distributions.
-Chapter 8 identifies certain Gaps that currently exist and plans on how
-to address them (for example, resources autoscaling).
-
-Terminology
------------
-
-General terminology definitions can be found in
-:doc:`cntt:common/glossary` and specific terms relating
-to this reference architecture are to be found in OpenStack Related
-Terminology :ref:`cntt:common/glossary:openstack related terminology`.
-
-Conventions
------------
-
-The key words "must", "must not", "required", "shall", "shall not",
-"should", "should not", "recommended", "may", and "optional"
-in this document are to be interpreted as described in
-RFC 2119 :cite:p:`rfc2119`.
-
-References
-----------
-
-.. bibliography::
-   :cited:
+Introduction
+============
+
+Overview
+--------
+
+This Reference Architecture is focussed on OpenStack as the Virtualised
+Infrastructure Manager (VIM) chosen based on the criteria laid out in
+the :doc:`ref_model:chapters/chapter01`.
+OpenStack :cite:p:`openstack` has the advantage of being a
+mature and widely accepted open-source technology; a strong ecosystem of
+vendors that support it, the OpenInfra Foundation for managing the
+community, and, most importantly, it is widely deployed by the global
+operator community for both internal infrastructure and external facing
+products and services. This means that the operators have existing staff
+with the right skill sets to support a Cloud Infrastructure
+(or Network Function Virtualisation Infrastructure, 

```

Comment:
There's a trailing whitespace here, would be good to remove.

Reply:
@ildikov Thanks! Removed